feat(curriculum): adding shape manager TS workshop (#63083)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-shape-manager/6915a8fed59e336bc937f5f2.md

---
id: 6915a8fed59e336bc937f5f2
title: Step 38
challengeType: 0
dashedName: step-38
---

# --description--

The last set of elements to select are the result elements. Use your `getElements` function to get the element with the ids `result-text` and `result-card`. Both should use `HTMLElement` as the type.

Remember you also created inline types `resultText` and `resultCard` for them, so you only need to reassign them.

# --hints--

> You should reassign the `resultText` variable with appropriate value.

Reviewer: ```suggestion
You should assign a value to the `resultText` variable.
```
